funnel/assets/js/utils/prism.js

```diff
@@ -0,0 +1,58 @@
+const PrismEmbed = {
+  activatePrism() {
+    $('code[class*=language-]:not(.activated):not(.activating)').each(
+      function activate() {
+        window.Prism.highlightElement(this);
+      }
+    );
+  },
+  hooked: false,
+  loadPrism() {
+    const CDN_CSS = 'https://unpkg.com/prismjs/themes/prism.min.css';
+    const CDN = [
+      'https://unpkg.com/prismjs/components/prism-core.min.js',
+      'https://unpkg.com/prismjs/plugins/autoloader/prism-autoloader.min.js',
```

Comment:
Shouldn't these URLs be in webpack or other dependency management system? They'll get scattered around code like this otherwise, and will be unauditable.